packages/babel-helper-replace-supers/src/index.ts

```diff
@@ -317,14 +327,19 @@ export default class ReplaceSupers {
 
     const handler = this.constantSuper ? looseHandlers : specHandlers;
 
-    memberExpressionToFunctions(this.methodPath, visitor, {
+    memberExpressionToFunctions<ReplaceState>(this.methodPath, visitor, {
       file: this.file,
       scope: this.methodPath.scope,
       isDerivedConstructor: this.isDerivedConstructor,
       isStatic: this.isStatic,
       isPrivateMethod: this.isPrivateMethod,
       getObjectRef: this.getObjectRef.bind(this),
       getSuperRef: this.getSuperRef.bind(this),
+      boundGet() {
+        // noop
+        // we dont need boundGet here, but memberExpressionToFunctions handler needs it.
+        throw new Error("");
```

Comment:
Throwing exception is definitely not noop. I am good with either make it an noop or throw an explicit error here.